Description: Use qemu on Ubuntu rather than qemu-system-i386
 See https://bugs.launchpad.net/ubuntu/+source/qemu-kvm/+bug/594888 for
 background.
Author: Colin Watson <cjwatson@ubuntu.com>
Forwarded: not-needed
Last-Update: 2010-06-16

Index: b/tests/util/grub-shell.in
===================================================================
--- a/tests/util/grub-shell.in
+++ b/tests/util/grub-shell.in
@@ -136,7 +136,7 @@
 cp ${isofile} ${fdafile}
 
 outfile=`mktemp`
-qemu-system-i386 ${qemuopts} -nographic -hda ${hdafile} -fda ${fdafile} -cdrom ${isofile} -boot ${bootdev} | tr -d "\r" >${outfile}
+qemu ${qemuopts} -nographic -hda ${hdafile} -fda ${fdafile} -cdrom ${isofile} -boot ${bootdev} | tr -d "\r" >${outfile}
 
 cat $outfile
 
